### DISCUSSION TITLE
Replication allocation logic: Whether it is a good idea to try not to distribute Replications on the same machine

### DISCUSSION THREAD

**Discussion by 7y-9** [seaweedfs/seaweedfs#5324] [Ideas]
Hi,

We followed the recommendations of the community and started many of dataNodes on the same machine and found that many replications were assigned to the same machine.

I have a half-baked idea about not distributing replications on the same machine.

Looking forward to hearing from the community.

**Comment by SmsS4** [comment]
You can use set `rack` flag in volume servers and set replication to 010

**Reply by 7y-9** [reply]
Thank you for your response. 
Indeed, we can utilize **010** to prevent multiple replication on a single machine. 
I would like to suggest that, even if it is **001**, we should strive to avoid multiple replication on one machine as much as possible.
